docs/content/latest/benchmark/tpcc-ysql/_index.md

```diff
@@ -53,44 +55,55 @@ You will need the IP addresses of the nodes in the cluster for the next step.
 {{< /tip>}}
 
 
-### Step 2. TPC-C Load Phase
-
-Before starting the workload, you will need to load the data first. Make sure
-to replace the IP addresses with that of the nodes in the cluster.
+### 2. Configure the benchmark
 
+If not working with the defaults, we can change the username, password, port, etc. using the configuration file at `config/workload_all.xml`. We can also change the terminals or the physical connections being used by the benchmark using the configuration.
 
 ```sh
-$ ./tpccbenchmark --create=true --load=true \
-  --nodes=127.0.0.1,127.0.0.2,127.0.0.3 \
-  --warehouses=100 \
-  --loaderthreads=48
+<dbtype>postgres</dbtype>
+<driver>org.postgresql.Driver</driver>
+<port>5433</5433>
+<username>yugabyte</username>
+<password></password>
+<isolation>TRANSACTION_REPEATABLE_READ</isolation>
+
+<terminals>100</terminals>
+<numDBConnections>10</numDBConnections>
 ```
 
 {{< note title="Note" >}}
-The default values for `nodes` is `127.0.0.1`, `warehouses` is `10` and
-`loaderthreads` is `10`.
+By default the number of terminals is 10 times the number of warehouses which is the max that the TPC-C spec allows. The number of DB connections is the same as the number of warehouses.
 {{< /note >}}
 
-### Step 3. TPC-C Execute Phase
-
-You can then run the workload against the database as follows:
-
-```sh
-$ ./tpccbenchmark --execute=true \
-  --nodes=127.0.0.1,127.0.0.2,127.0.0.3 \
-  --warehouses=100  \
-  -o outputfile
-```
+<ul class="nav nav-tabs nav-tabs-yb">
```

Comment:
This should be in a section: `3. Run the benchmark`

Reply:
That is not needed.
Since step 3 and step 4 are coming from the individual files.